examples/custom-server/server/index.ts

```diff
@@ -0,0 +1,27 @@
+import { createServer } from "http";
+import { parse } from "url";
+import { createBlitzApp } from "@blitzjs/server";
+
+const { PORT = "3000", NODE_ENV = "development" } = process.env
+
+const dev = NODE_ENV === "development"
+
+async function main() {
+  const { requestHandler } = await createBlitzApp({ dev })
+
+  createServer((req, res) => {
+    const parsedUrl = parse(req.url!, true)
+    const { pathname } = parsedUrl
+
+    if (pathname === "/hello") {
```

Comment:
🙋‍♂️ Question here

Would there be a way to generating/accessing blitz context for requests here? Would not be useful for http-methods like here as one would just write them as API-functions, but if one instead did a WebSocket-server it would be useful to authenticate and access blitz context within this.

Reply:
Yes, one can do `const session = await getSessionContext(req, res)` here to get the session.